libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -95,11 +95,17 @@ void AP_Parachute::enabled(bool on_off)
     AP::logger().Write_Event(_enabled ? LogEvent::PARACHUTE_ENABLED : LogEvent::PARACHUTE_DISABLED);
 }
 
+/// enabled_by_aux - enable or disable parachute release using aux switch
+void AP_Parachute::enabled_by_aux(bool on_off)
+{
+    _enabled_by_aux = on_off;
+}
+
 /// release - release parachute
 void AP_Parachute::release()
 {
     // exit immediately if not enabled
-    if (_enabled <= 0) {
+    if (_enabled <= 0 || _enabled_by_aux <=0) {
```

Comment:
```suggestion
    if (_enabled <= 0 || !_enabled_by_aux) {
```
no need to compare with a bool

Reply:
oops! Yeah, I have fixed this. Thanks for pointing it out :-)